ddtrace/internal/writer.py

```diff
@@ -435,6 +443,9 @@ def write(self, spans=None):
             )
             self._metrics_dist("buffer.dropped.traces", 1, tags=["reason:full"])
             self._metrics_dist("buffer.dropped.bytes", payload_size, tags=["reason:full"])
+        except Exception:
```

Comment:
which exceptions do we expect `encoder.put(span)` to raise? are there any more specific exceptions we should keep an eye out for?

Reply:
We retry on these errors: compat.httplib.HTTPException, OSError, IOError. This case would catch all other errors. I am not sure what else could be raised but this missing case is probably why we have dropped_traces > accepted_traces